### ISSUE TITLE
There is no UI/UX hint that a new organisation should be granted an organisation-level indemnification

### ISSUE THREAD

**Issue by Darwinkel** [minvws/nl-kat-rocky#168]
**Is your feature request related to a problem? Please describe.**
See title. Since OpenKAT v1.5.

Any newly created organisation does not automatically have an organisation-level indemnification. This is not clearly hinted/communicated to admins. The indemnification must be added before clearance levels for all users of the organisation can be changed.

**Describe the solution you'd like**
There should be a hint or message that an indemnification should be added to the organisation.

**Describe alternatives you've considered**
I'm not sure there are any alternative options, short of integrating this step into the "Create Organisation" flow.

**Additional context**
None.

**Comment by HeleenSG** [comment]
I would expect this to be part of the setup flow for the organisation. Is there a reason why it currently isn't? If not adding it there would make sense.

**Comment by underdarknl** [comment]
True, but not all organizations are created via the setup flow, Some are created by Admins via de django admin, or via the Json api's. If no indemnification is present we should communicate this to users on pages where they would expect to set indemnificatoins / start boefjes.

**Comment by psy0rz** [comment]
#249 fixes this i think?
